packages/administration/src/Resources/views/crud/inline/actions.html.twig

```diff
@@ -1,10 +1,12 @@
 {% macro render(actions) %}
     <div class="wrap-bar__buttons">
         {% for action in actions %}
-            <a href="{{ action.url }}" class="btn btn--plus wrap-bar__btn {{ action.cssClass }}">
+            <a href="{{ action.url }}"
+               class="btn wrap-bar__btn {{ action.cssClass }}"
+               {% if action.openInNewTab %}target="_blank"{% endif %}
+            >
                 {% if action.icon is not null %}
-                    {# TODO: Replace by defined icon #}
-                    <i class="btn__icon">+</i>
+                    <i class="btn__icon svg svg-{{ action.icon }}"></i>
```

Comment:
needs to be updated now after https://github.com/shopsys/shopsys/pull/3765 is merged

Reply:
updated